vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -405,15 +414,22 @@ void Publisher::OnSetPublisherExclude(
     ledger::ResultCallback callback) {
   if (result != ledger::Result::LEDGER_OK &&
       result != ledger::Result::NOT_FOUND) {
+    BLOG(0, "Publisher exclude status not saved");
     callback(result);
     return;
   }
 
-  if (!publisher_info || publisher_info->excluded == exclude) {
+  if (!publisher_info) {
+    BLOG(0, "Publisher is null");
     callback(ledger::Result::LEDGER_ERROR);
     return;
   }
 
+  if (publisher_info->excluded == exclude) {
+    callback(ledger::Result::LEDGER_OK);
```

Comment:
Before we returned `ledger::Result::LEDGER_ERROR` for this condition?

Reply:
correct, but it doesn't make sense to return error if status is the same, as we don't want to alert user for it